PostSharp.Community.StructuralEquality.Tests/Fody/IntegrationTests/EqualsTests.cs

```diff
@@ -118,12 +120,19 @@ bool CheckEqualityOnTypesForTypeCheck(string left, string right)
         [InlineData("EqualsOrSubtypeSubClass", "EqualsOrSubtypeClass", true)]
         [InlineData("EqualsOrSubtypeSubClass", "EqualsOrSubtypeSubClass", true)]
         [InlineData("ExactlyOfTypeClass", "ExactlyOfTypeClass", true)]
-        [InlineData("ExactlyOfTypeSubClass", "ExactlyOfTypeClass", false)]
-        [InlineData("ExactlyOfTypeClass", "ExactlyOfTypeSubClass", true)]
+        [InlineData("ExactlyOfTypeSubClass", "ExactlyOfTypeClass", true)]
+        [InlineData("ExactlyOfTypeClass", "ExactlyOfTypeSubClass", false)]
         [InlineData("ExactlyOfTypeSubClass", "ExactlyOfTypeSubClass", false)]
         [InlineData("ExactlyTheSameTypeAsThisClass", "ExactlyTheSameTypeAsThisClass", true)]
         [InlineData("ExactlyTheSameTypeAsThisClass", "ExactlyTheSameTypeAsThisSubClass", false)]
         [InlineData("ExactlyTheSameTypeAsThisSubClass", "ExactlyTheSameTypeAsThisClass", false)]
+        
+        [InlineData("SameTypeOrSubtypeStruct", "SameTypeOrSubtypeStruct", true)]
+        [InlineData("ExactlyOfTypeStruct", "ExactlyOfTypeStruct", true)]
+        [InlineData("ExactlyTheSameTypeAsThisStruct", "ExactlyTheSameTypeAsThisStruct", true)]
+        [InlineData("SameTypeOrSubtypeStruct", "ExactlyOfTypeStruct", false)]
+        [InlineData("ExactlyOfTypeStruct", "SameTypeOrSubtypeStruct", false)]
+        [InlineData("ExactlyTheSameTypeAsThisStruct", "ExactlyOfTypeStruct", false)]
         //TODO: support sub classes
```

Comment:
I don't understand what this TODO means.

Reply:
That's not our code.